src/main/java/frc/robot/commands/vision/TestVisionDistance.java

```diff
@@ -60,15 +62,20 @@ public void execute(){
       // reset the timer
       m_endTimer.reset();
       // If kPrintDelay seconds have passed, print the data
-      // TODO: Consider using shuffleboard instead of printing
+      double driveDistance = m_drive.getPose().getTranslation().getDistance(m_driveStartTranslation);
+      double visionDistance = m_currentPose.getTranslation().getDistance(m_visionStartTranslation);
       if (m_printTimer.advanceIfElapsed(kPrintDelay)) {
-        double driveDistance = m_drive.getPose().getTranslation().getDistance(m_driveStartTranslation);
-        double visionDistance = m_currentPose.getTranslation().getDistance(m_visionStartTranslation);
         System.out.printf("\nEncoder distance: %.4f\nVision distance: %.4f\n",
           driveDistance, visionDistance);
         System.out.printf("Difference: %.4f\nPercent difference: %.4f%%\n",
           visionDistance - driveDistance, (visionDistance - driveDistance) / driveDistance * 100);
       } 
+      if(Constants.kLogging){
+        LogManager.addDoubleArray("Vision/DistanceTest", new double[]{
+          driveDistance, visionDistance, 
+          visionDistance - driveDistance, (visionDistance - driveDistance) / driveDistance * 100
+        });
+      }
```

Comment:
perhaps it is a good idea to log this in the drivetrain rather than in this spicific test. I know it would be longging in this test but I think if you are already printing it out it is prorobly good enough for what your are doing then again it dosen't really hurt to leave it here. 

TLDR; add this to drivetrain not just here.

Reply:
This is the distance from vision and encoders, and is specific to this test. I kept both the print statements and this because print statements are easier to access (DriverStation logs), but logging could be helpful to see all of the data. It also logs constantly instead of every second, which could give better data. I think the pose is already logged in drivetrain, so I'm not sure what this would do. It measures distance from the starting point in this test, and we don't really need distance from the actual starting point to wherever the robot is on the field. This test also disables vision, which we should not do on the actual robot during SFR.

Reply:
I was more thinking the distance between the apirl tag and the camera or robot.